tests/rootless.sh

```diff
@@ -55,6 +55,21 @@ function disable_idmap() {
 	[ -e /usr/bin/newgidmap ] && mv /usr/bin/{,unused-}newgidmap
 }
 
+# FEATURE: Opportunistic cgroups support, allowing a rootless container to set
+#          resource limits on condition that cgroupsPath is set to a path the
+#          rootless user has permissions on.
+
+function enable_cgroup() {
+	# Set up cgroups for use in rootless containers.
+	mkdir -p /sys/fs/cgroup/{blkio,cpu,cpuacct,cpuset,devices,freezer,hugetlb,memory,net_cls,net_prio,openrc,perf_event,pids,systemd}/runc-cgroups-integration-test
+	chown rootless:rootless -R /sys/fs/cgroup/{blkio,cpu,cpuacct,cpuset,devices,freezer,hugetlb,memory,net_cls,net_prio,openrc,perf_event,pids,systemd}/runc-cgroups-integration-test
```

Comment:
I would prefer if we did `chown rootless:rootless -R /sys/fs/cgroup/*/runc-cgroups-integration-test` here.